webapp/portlet/src/main/resources/locale/portlet/Portlets_en.properties

```
@@ -749,3 +749,12 @@ attachment.unauthorizedAccess=Unauthorized operation
 attachment.objectNotFound=Error while accessing data. Please refresh the page and try again.
 attachment.attachmentNotFound=Attachment not found
 attachment.fileReadingError=Error while reading file content
+
+#####################################################################################
+#                              Portlet Banner                                   #
+#####################################################################################
+
+portletBanner.title.BannerUpdated=Background successfully updated
+portletBanner.title.BannerDeleted=Background successfully deleted
+portletBanner.title.deleteBanner=Delete Background
+portletBanner.title.ChangeBanner=Change background
```

```suggestion
portletBanner.title.BannerUpdated=Background successfully updated
banner.title.BannerDeleted=Background successfully deleted
banner.title.deleteBanner=Delete Background
banner.title.ChangeBanner=Change background
```

Translations to be verified with O